pkgs/top-level/python-packages.nix

```diff
@@ -6636,6 +6636,12 @@ in {
 
   uvicorn = callPackage ../development/python-modules/uvicorn { };
 
+  compiledb = callPackage ../development/python-modules/compiledb { };
```

Comment:
Please make these be inserted in alphabetical order in the file

Reply:
it cannot be done, the file isn't sorted

Reply:
It is "somewhat" sorted. We're basically trying to not make it any worse. I think eventually `nixpkgs-fmt` will be able to help us here.

Reply:
@costrouc done, thanks for the review!